docs/support/plugin-support.md

```diff
@@ -64,6 +64,6 @@ For example, a plugin to provide a formatter which has itself been abandoned has
 | `hls-overloaded-record-dot-plugin`  | 2    |                          |
 | `hls-semantic-tokens-plugin`        | 2    |                          |
 | `hls-floskell-plugin`               | 3    |                          |
-| `hls-stan-plugin`                   | 3    | 9.2.(4-8)                |
+| `hls-stan-plugin`                   | 3    | 9.2.(4-8), 9.10.1        |
 | `hls-retrie-plugin`                 | 3    |                          |
```

Comment:
There are more plugins not supported on 9.10!

Reply:
Which ones? In `haskell-language-server.cabal`, I could only find stan having:

```
common stan
  if flag(stan) && (impl(ghc > 8.8.1) && impl(ghc <= 9.2.3) || impl(ghc >= 9.4.0) && impl(ghc < 9.10.0))
    build-depends: haskell-language-server:hls-stan-plugin
    cpp-options: -Dhls_stan

...

  if flag(stan) && (impl(ghc > 8.8.1) && impl(ghc <= 9.2.3) || impl(ghc >= 9.4.0) && impl(ghc < 9.10.0))
    buildable: True
  else
    buildable: False
```

Reply:
Ah, I found more in `.github/workflow/test.yml`

Reply:
Hmm, I'm surprised nothing else has guards in the cabal file? I would expect us to still try and build them if not?

Reply:
Does that work? Like will this not affect `cabal install haskell-language-server -w 9.10.1`?

Reply:
I'm unsure what our policy should be here. We should probably either be consistently setting things in `cabal.project` or in the cabal file...